lib/galaxy/tool_util/linters/outputs.py

```diff
@@ -45,7 +45,14 @@ def lint_output(tool_xml, lint_ctx):
 
         label = output.attrib.get("label", "${tool.name} on ${on_string}")
         if label in labels:
-            lint_ctx.error(f"Tool output [{name}] uses duplicated label '{label}'", node=output)
+            filter_node = output.find(".//filter")
+            if filter_node is not None:
+                lint_ctx.info(
+                    f"Tool output [{name}] uses duplicated label '{label}', double check if filters imply disjoint cases",
```

Comment:
> Then it might OK to allow for the same label. For instance if `from_work_dir` needs to be different when a flag is set or not set.

I'd move the output to the right place or use the same output name, which is much friendlier for predicting what outputs will be generated. I think a warning is just the right level here.

Reply:
If we make this a warning then the outcome (IUC wise) would be the same (https://github.com/galaxyproject/tools-iuc/pull/5260), ie. the PR workflow would fail. 

But, given your idea to skip linting for the push to master/main we could just raise the fail level to error for the merge. 

Just tell me your preference (warning / info) and I will change it.

Reply:
How about this example @mvdbeek: https://github.com/galaxyproteomics/tools-galaxyp/pull/721 ?

Reply:
I'm sorry, can you point out what I should look at there ?

Reply:
There is an output dataset and an output collection  which are mutually exclusive. They have the same name which I find acceptable in this case.

Reply:
Yes, according to my intuition of the meaning of "warning".